src/PiWeb.Import.Sdk/Modules/ImportFormat/IImportFormat.cs

```diff
@@ -56,5 +56,11 @@ public interface IImportFormat
     /// <param name="context">Provides information about the import context.</param>
     Task<ImportData> ParseImportData(IImportGroup importGroup, IParseContext context);
 
+    /// <summary>
+    /// Gets the import format configuration specifying supported import format configuration settings.
+    /// </summary>
+    /// <returns>The import format configuration.</returns>
+    ImportFormatConfiguration GetImportFormatConfiguration();
```

Comment:
We are in IImportFormat, so we can ommit the ImportFromat here. Instead I think we should suggest that this is the default configuration which is only used when the user does not make any other configurations. So GetDefaultConfiguration?

Reply:
Done